README.md

```diff
@@ -46,8 +46,8 @@ For Chrome:
 For Firefox
 
 1. Open Firefox
-1. Click Preferences -> Extensions & Themes 
-1. Click on the cog icon -> Install Add-on From File -> Select the `gitpod.xpi` file
+1. Go to `about:debugging#/runtime/this-firefox`
+1. Click Load Temporary Add-on -> Select the `gitpod.xpi` file
```

Comment:
**small nit (non-blocking)**:

```suggestion
1. Go to `about:debugging#/runtime/this-firefox`
1. Click Load Temporary Add-on -> Select the `gitpod.xpi` file
```
```suggestion
- Go to `about:debugging#/runtime/this-firefox`
- Click Load Temporary Add-on -> Select the `gitpod.xpi` file
```

Reply:
numeric steps seem reasonable in this case, why do you prefer list items instead?

also, other blocks in this readme use numeric steps, we'd need to change them all to be consistent

Reply:
to release this fix I will be merging, but if you feel strong about markdown changes, please raise a PR to update the whole README, thanks for reviewing 🙏